p02_3.py

```diff
@@ -0,0 +1,30 @@
+def mean(arr):
+	sum = 0
+	l = len(arr)
+	if l == 0:
+		print("There are no numbers in the array!")
+		return
```

Comment:
Should either raise (see above) or return 0 (mean of nothing is 0). I prefer to raise though